web/src/main/webapp/script.js

```diff
@@ -12,6 +12,48 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function searchMe() {
+  var input, filter, ul, li, item, i, txtValue;
+  input = document.getElementById("myInput");
+  filter = input.value.toUpperCase();
+  ul = document.getElementById("task-list");
+  li = ul.getElementsByTagName("li"); 
+   for (i = 0; i < li.length; i++) {
+    item = li[i];
+    txtValue = item.textContent || item.innerText;
+    if (txtValue.toUpperCase().indexOf(filter) > -1) {
+      li[i].style.display = "";
+    } else {
+      li[i].style.display = "none";
+    }
+  }
+}
+
+function loadTasks() {
+  fetch('/list-stock').then(response => response.json()).then((stocks) => {
+    const taskListElement = document.getElementById('task-list');
+    stocks.forEach((stock) => {
+      taskListElement.appendChild(createTaskElement(stock));
+    })
+  });
+  priceElement.innerText = task.price;
+}
+
+/** Creates an element that represents a task, including its delete button. */
+function createTaskElement(task) {
+  const taskElement = document.createElement('li');
+  taskElement.className = 'task';
+
+  const titleElement = document.createElement('span');
```

Comment:
No action required: This line `const titleElement = document.createElement('span');` is a great example of a good variable name.